Homework2/Program.cs

```diff
@@ -10,6 +10,74 @@ class Program
     {
         static void Main(string[] args)
         {
+            int nominal = 0;
```

Comment:
It's a good practice to introduce new variables as close to the place where they are used. As example, you use `nominal` first the scope of do-while loop on the 23rd line. So you can declare the variable before just the loop. And the same for all the other variables.